docs/content/docs/connectors/datastream/dynamodb.md

```diff
@@ -141,7 +268,7 @@ annotations see [here](https://docs.aws.amazon.com/sdk-for-java/latest/developer
 
 A sample application using a custom `ElementConverter` can be found [here](https://github.com/apache/flink-connector-aws/blob/main/flink-connector-aws/flink-connector-dynamodb/src/test/java/org/apache/flink/connector/dynamodb/sink/examples/SinkIntoDynamoDb.java). A sample application using the `DynamoDbBeanElementConverter` can be found [here](https://github.com/apache/flink-connector-aws/blob/main/flink-connector-aws/flink-connector-dynamodb/src/test/java/org/apache/flink/connector/dynamodb/sink/examples/SinkDynamoDbBeanIntoDynamoDb.java).
```

Comment:
I would show a snippet of the converter's `apply()` method, using something more realistic than the linked example.

In the example the input record is a `Long`. In reality it would be a POJO.
In the example the partitionKey and sortKey are random. In reality they would be mapped from fields of the POJO.

Reply:
How are  partition_key and sort_key mapped also determines whether the sink will do upsert or append.
I we assume the reader knows DDB we can skip this. Otherwise, we should explain

Reply:
Let's refactor the sink docs as part of a different story!